### ISSUE TITLE
The CR "installations" deployed by tigera-operator is missing "conditions" as part of the status

### ISSUE THREAD

**Issue by sirajyasin** [tigera/operator#1055]
The CR **installations** deployed by tigera-operator is missing "conditions" as part of the status. This creates issue verifying the resources applied. As an example the [cli-utils](https://github.com/kubernetes-sigs/cli-utils) status poller (through applier) fails to fetch status of this resource due to [condition](https://github.com/kubernetes-sigs/cli-utils/tree/master/pkg/kstatus#conditions) not satisfied. 
I can understand that the status of calico installation can be tracked through tigerastatus CR, but then is it possible to have "conditions" added for installations for consistency.

```
kubectl get installations -o json | jq -r ".items[0].status"
{
  "mtu": 1440,
  "variant": "Calico"
}
```
## Expected Behavior
"conditions" should be available as part of the status

## Current Behavior

Status has just mtu and variant and missing "conditions"

## Possible Solution

add "conditions" to status in addition to mtu and variant

## Steps to Reproduce (for bugs)

1. Deploy tigera-operator
2. kubectl get installations -o json | jq -r ".items[0].status"

## Context
This creates issue verifying the resources applied.

## Your Environment

* Operating System and version:
* Link to your project (optional):

**Comment by caseydavenport** [comment]
Seems like we already have the information available to implement the "Ready" condition in use for the TigeraStatus, so we can mirror that into conditions in the respective CRs. Anyone interested in taking a stab at this?

**Comment by tmjd** [comment]
I believe this has been addressed with #1954 and is available in the latest Calico release.
